### DISCUSSION TITLE
Widget Properties not shown in java SWT Dialog Design view

### DISCUSSION THREAD

**Discussion by ffedeli** [eclipse/windowbuilder#759] [Q&A]
After updating WindowsBuilder to 1.16.0 Milestone, widget property list (in java SWT Dialogs Design view) contains no elements, it's completely empty.

Eclipse Version: 2023-12 (4.30.0) Build id: 20231201-2043

![Screenshot 2024-05-07 164314](https://github.com/eclipse-windowbuilder/windowbuilder/assets/34574583/42d053df-8f1f-4624-b66f-d4f487a2717c)

**Comment by ptziegler** [comment]
From a clean 4.31 with WindowBuilder M2:

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/5ac7cb0d-5bbf-42d0-a4ab-bf1362fd9c78)

What version of GEF is installied in your Eclipse? The property table depends on the class that has only been added in the latest milestone.

Can you check if anything is logged in the error view?
